demo/js/components/victory-voronoi-container-demo.js

```diff
@@ -109,7 +109,7 @@ class App extends React.Component {
               <VictoryVoronoiContainer
                 labels={({ datum }) => `I'm kind of a long label ${datum.y}`}
                 mouseFollowTooltips
-                labelComponent={<VictoryTooltip constrainToVisibleArea tooltipWidth={80} />}
```

Comment:
I think this was meant to be `flyoutWidth`

Reply:
got it. i will fix this.